docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -117,9 +117,68 @@ vaccination campaign.
 2. This model uses country-level data, and cannot be used to model local measles
 outbreaks due to lack of vaccination in small communities.
 
+Some of the **assumptions** made in this model are:
```

Comment:
You might also add some of the assumptions that Natalie told us about---I don't remember all the specifics, but I believe in their nonfatal model they assumed that 95% of unvaccinated populations, globally, would be incident cases of measles? And I know this is in your states data table, but it also seems worth stressing that they assume a 50/50 split of moderate/severe measles in their estimates

Reply:
@beatrixkh Thanks for your comments. But I think 95% attack rate assumption will be too confusing as this documentation is intended for all kinds of audience. We have already cited YLD Appendix and it is mentioned there. But I am open to discussion and if everyone think its important, I will add that.

Reply:
I will include 50/50 split assumption. Thanks for mentioning that.